InventorySales/composer.json

```diff
@@ -19,7 +19,8 @@
         "magento/module-sales": "*"
     },
     "suggest": {
-        "magento/module-inventory-catalog": "*"
+        "magento/module-inventory-catalog": "*",
+        "magento/module-quote": "*"
```

Comment:
It's not a good idea to add new dependencies

Reply:
Can you advise where is the best place to move this plugin? We have two dependency in the plugin - sales and quote.